### ISSUE TITLE
Bank account total balance always 0€

### ISSUE THREAD

**Issue by abilbaotm** [Dolibarr/dolibarr#3875]
![image](https://cloud.githubusercontent.com/assets/12185014/10921971/e373cf4c-8278-11e5-8373-cad95d27bdc9.png)
Dolibarr 3.8.1
/compta/bank/account.php

**Comment by abilbaotm** [comment]
Fixed! /compta/bank/account.php @ line 955 

```
        print '<td align="right" class="nowrap"><b>'.price($solde, 0, $langs, 0, 0, -1, $object->currency_code).'</b></td>';
```

by

```
        print '<td align="right" class="nowrap"><b>'.price($total, 0, $langs, 0, 0, -1, $object->currency_code).'</b></td>';
```

**Comment by marcosgdf** [comment]
Are you sure `$solde` is the same as `$total`?

**Comment by abilbaotm** [comment]
In older versions the line is set with $total and if you change the bug is fixed. I don't know why had been changed at 3.8.1

**Comment by rdoursenaud** [comment]
Fixed by 334c14d088c0f22805247917770dbcc5f6a8e3f8. Will be available in next release.
